CHANGES_UNRELEASED.md

```diff
@@ -18,4 +18,9 @@ Use the template below to make assigning a version number during the release cut
   - Description of the change with a link to the pull request ([#0000](https://github.com/mozilla/application-services/pull/0000))
 
 -->
-- `./libs/build-all.sh` now displays a more helpful error message when a file fails checksum integrity test.
+
+## fxa-client
+  - The old StateV1 persisted state schema is now removed. ([#4218](https://github.com/mozilla/application-services/pull/4218))
```

Comment:
Let's add a brief comment here about what this change means for consumers, both because it's useful info for consumers and because it helps us reason through the implications.

In practice I expect it's something like "Users on very old versions of this component will no longer be able to cleanly update to this new version. They will...???" <- actually I'm not 100% sure what will happen to those users if they do try to update, let's see if we can work it out :-)